src/crypto/crypto_hash.cc

```diff
@@ -144,12 +145,10 @@ void Hash::GetCachedAliases(const FunctionCallbackInfo<Value>& args) {
   Isolate* isolate = args.GetIsolate();
   Local<Context> context = args.GetIsolate()->GetCurrentContext();
   Environment* env = Environment::GetCurrent(context);
-  std::vector<Local<Name>> names;
-  std::vector<Local<Value>> values;
   size_t size = env->alias_to_md_id_map.size();
+  LocalVector<Name> names(isolate, size);
+  LocalVector<Value> values(isolate, size);
```

Comment:
I've been bitten by this too. You cannot just replace `reserve(size)` with an initialization that contains `size`. It will create a vector filled with default-initialized values (in this case empty `Local`s). Then `push_back` will add more elements to it.